services/app-api/forms/sar.json

```diff
@@ -859,6 +761,13 @@
                   "rule": "targetPopulations"
                 }
               },
+              {
+                "id": "ret-mpdprp-divider-other",
+                "type": "sectionHeader",
+                "props": {
+                  "divider": "top"
+                }
+              },
```

Comment:
if RET is the only section where we are adding these, could we make it part of the SectionHeader component rather than a form-based prop? Is the challenge getting it only between sections and not on all of them?

Reply:
@gmrabian I'd need another pair of eyes on this one. The dividers should only be after the first section, provided more than one section exists.

Reply:
Looking at this now.
Tagging @jessabean who I'm sure would love to find a CSS solution here :)

Reply:
sibling selectors are your friends here!!! Example:

```css
tr + tr {
  border-top: 1px solid green
}
```

☝🏻 that would add a green border at the top of the second tr, whenever it is a sibling of another tr. I haven't looked deep at the code yet, but that's the first place I'd look to accomplish this. I can try to get a more detailed suggestion once I'm done reviewing!

Reply:
Looping back: this won't work for the way the content is currently structured. So I think Garrett's suggestion is the way to do. Ideally, each "section" would actually be a `<fieldset>` with a `<legend>` as the heading, but it is not feasible to do that here...it would be a big refactor + headache.